staging/publishing/rules.yaml

```diff
@@ -1281,6 +1285,10 @@ rules:
       branch: master
     - repository: cloud-provider
       branch: master
+    - repository: component-base
+      branch: master
+    - repository: kube-controller-manager
```

Comment:
Btw, this was one of the reasons I initially moved the versioned service controller config to `k8s.io/cloud-provider`. Now anything that imports the cloud-provider has to import kube-controller-manager. I'll have a follow-up PR to fix this with the validation/generator wired in correctly.

Reply:
I get that KCM has to pull in the cloud provider controllers but we should be able to break that when the extraction effort finishes. Having to pull in the KCM because you pulled in the cloud-provider is definitely wrong. That's not something we want to do with the KAS or the Kubelet. If we can't fix it with your wiring change, then I think we would be better of moving them to the new controller-manager repo for now. I agree that these controllers are generic cloud-provider controllers but ...

Reply:
We wouldn't have to rely on `kube-controller-manager` once the controller manager config types move to `k8s.io/controller-manager`. That seems like the proper fix here as opposed to moving all the controllers to controller-manager.

Reply:
I see the PR for controller-manager got moved, I can take a stab at moving those here.

Reply:
Or I think @cici37 had a PR to move them. We should either block this PR on that or merge this as is and fix it once that PR is merged.